solenoid_app/forms.py

```diff
@@ -8,4 +8,15 @@ class DataSetForm(forms.Form):
     x = forms.FloatField()
     force = forms.FloatField()
     awg = forms.CharField()
-    compute = forms.CharField()
\ No newline at end of file
+    compute = forms.CharField()
+
+class GraphForm(forms.Form):
+    voltage = forms.FloatField()
+    length = forms.FloatField()
+    r_not = forms.FloatField()
+    r_a = forms.FloatField()
+    x = forms.FloatField()
+    force = forms.FloatField()
+    awg = forms.CharField()
+    compute = forms.CharField()
+    toGraph = forms.CharField()
```

Comment:
Since it looks like the only difference here is toGraph, would it make more sense for this to just inherit from the DataSetForm?

Reply:
+1